waffle_configs/bedrock-prod.env

```diff
@@ -25,13 +25,12 @@ SWITCH_EXPERIMENT_FIREFOX_NEW_TRAILHEAD=off
 SWITCH_EXPERIMENT_FIREFOX_WHATSNEW_70=off
 SWITCH_FIREFOX_ELECTION_BUILD=on
 SWITCH_FIREFOX_DEV_FALL=on
-SWITCH_FIREFOX_CONCERT_SERIES=on
```

Comment:
Deleted in bedrock with this PR: https://github.com/mozilla/bedrock/pull/8507/files#diff-784e0d4de42af29595e70315a6be5337L867